packages/app-mining/src/components/ClaimAll.tsx

```diff
@@ -225,7 +278,7 @@ const ClaimAll = (props: BlockProps) => {
           >
             <PhalaStakePoolTransactionFeeLabel action={extrinsic} />
             <ModalButton
-              disabled={!address || isAddressError}
+              disabled={!address || isAddressError || confirmLock}
```

Comment:
Maybe it's more intuitive to use `loading` prop when confirming.

Reply:
notion/pd says use disability